src/main/java/games/coltexpress/ColtExpressForwardModel.java

```diff
@@ -60,12 +61,11 @@ public void _setup(AbstractGameState firstState) {
             cegs.playerDecks.add(playerCards);
             playerCards.shuffle(new Random(cep.getRandomSeed()+playerIndex));
 
-            Deck<ColtExpressCard> playerHand = new Deck<>(
-                    "playerHand" + playerIndex, playerIndex);
+            Deck<ColtExpressCard> playerHand = new Deck<>("playerHand" + playerIndex, playerIndex, VisibilityMode.VISIBLE_TO_OWNER);
 
             cegs.playerHandCards.add(playerHand);
 
-            Deck<Loot> loot = new Deck<>("playerLoot" + playerIndex, playerIndex);
+            Deck<Loot> loot = new Deck<>("playerLoot" + playerIndex, playerIndex, VisibilityMode.VISIBLE_TO_OWNER);
```

Comment:
This is actually not visible to the owner either, hidden to all. No one knows what loot values they collected until the end of the game. Although I see later that this means to distinguish between types of loot while loot values should be kept hidden. So I guess this doesn't change much the logic?

Reply:
Oh! Whenever we play Colt Express we always look at the loot tiles we pick up! Ooops. (which is why I used VISIBLE_TO_OWNER)

But...the main substantive point is that you do know the important stuff (just not which loot bags are 250 versus 500). Actually, thinking about this, this should probably be VISIBLE_TO_ALL on this logic.